src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue26857.cs

```diff
@@ -0,0 +1,26 @@
+
+using NUnit.Framework;
+using NUnit.Framework.Legacy;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue26857 : _IssuesUITest
+	{
+		public override string Issue => "ListView ScrollTo position always remains at the start even when set to Center or End";
+
+		public Issue26857(TestDevice device) : base(device)
+		{
+		}
+
+		[Test]
+		[Category(UITestCategories.ListView)]
+		public void Issue26857Test_SelectItem()
+		{
+			App.WaitForElement("Button");
+			App.Tap("Button");
+			VerifyScreenshot();
```

Comment:
Can we extend this test to check all positions? Start, Center and End?

Reply:
@mattleibow, I have extended the test to cover all positions: Start, Center, and End. Could you please review it?

Reply:
Looks amazing!